parlai/agents/bert_classifier/bert_classifier.py

```diff
@@ -90,20 +99,6 @@ def add_cmdline_args(
         """
         super().add_cmdline_args(parser, partial_opt=partial_opt)
         parser = parser.add_argument_group("BERT Classifier Arguments")
-        parser.add_argument(
```

Comment:
was this option just never used?

Reply:
Right. I actually also found it in BertWrapper's add_common_args function, also never used, so I'll remove it from there